pkg/build/info.proto

```diff
@@ -25,7 +25,7 @@ message Info {
   optional string go_version = 1 [(gogoproto.nullable) = false];
   optional string tag = 2 [(gogoproto.nullable) = false];
   optional string time = 3 [(gogoproto.nullable) = false];
-  optional string dependencies = 4 [(gogoproto.nullable) = false];
+  optional string revision = 4 [(gogoproto.nullable) = false];
```

Comment:
this is more of a rename. We used to list all dependencies (when we had those), and before this PR, this is the `vendor` sha. This is now the repo sha, so a more fitting name.